setup.py

```diff
@@ -24,18 +24,19 @@
     license="MIT",
     packages=find_packages(),
     package_data={
-        "sphinx_panels": ["css/panels.css", "css/bs-cards.css", "css/bs-grids.css"]
+        "sphinx_panels": ["css/bs-cards.css", "css/bs-grids.css", "css/bs-borders.css"]
```

Comment:
I take it that these are straight from the bootstrap bundle?

Reply:
yes all the css they contain comes from `boostrap.min.css`, but split into loosely the functionality they apply to

Reply:
gotcha - so if bootstrap updates itself and we want those updates, we need to find the right part of `.min.css`, then copy paste those sections and put them in these files?